ERROR: Cannot find module 'mineflayer/lib/iterators'

When I try to run the code it just throws this error:
```js
node:internal/modules/cjs/loader:928
  throw err;
  ^

Error: Cannot find module 'mineflayer/lib/iterators'
Require stack:
- E:\Node Bots\Temp\node_modules\mineflayer-baritone\utils.js
- E:\Node Bots\Temp\node_modules\mineflayer-baritone\goals.js
- E:\Node Bots\Temp\node_modules\mineflayer-baritone\index.js
- E:\Node Bots\Temp\baritone.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (E:\Node Bots\Temp\node_modules\mineflayer-baritone\utils.js:1:29)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:997:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\Node Bots\\Temp\\node_modules\\mineflayer-baritone\\utils.js',
    'E:\\Node Bots\\Temp\\node_modules\\mineflayer-baritone\\goals.js',
    'E:\\Node Bots\\Temp\\node_modules\\mineflayer-baritone\\index.js',
    'E:\\Node Bots\\Temp\\baritone.js'
  ]
}
```

--- a/utils.js
+++ b/utils.js
@@ -1,4 +1,4 @@
-const { RaycastIterator } = require('mineflayer/lib/iterators')
+const { RaycastIterator } = require('prismarine-world').iterators
 
 function isPlayerOnBlock(playerPosition, blockPosition, onGround=false, overhang=true) {
 	// returns true if you can stand on the block

